x-pack/plugins/observability/public/plugin.ts

```diff
@@ -141,6 +141,19 @@ export class Plugin
         defaultMessage: 'Overview',
       }),
       updater$: appUpdater$,
+      keywords: [
+        'observability',
+        'monitor',
+        'logs',
+        'metrics',
+        'apm',
+        'performance',
+        'trace',
+        'agent',
+        'rum',
+        'user',
+        'experience',
```

Comment:
What does it mean? The user can type any of these keywords and we'd show the Observability option?

Reply:
correct